templates/compositions/ec-component-content-item/content-item.html.twig

```diff
@@ -0,0 +1,92 @@
+{% spaceless %}
+
+{#
+  /**
+   * @file
+   * Theme implementation for a ECL Content Item.
+   */
+  Parameters:
+    - "content_item" (associative array) (default: predefined structure): format:
+        {
+          description: '', (string) (default: ''): Description of Content Item
+          meta: [], (array) (default: []): Meta information for the Content Item
+          title: {}, (associative array) (default: {}): Predefined structure compatible with Link component. If Content Item type is a 'tile', only label property is required.
+          image: '', (string) (default: ''): Url/path to background image
+          image_position: 'after' (string) (Can be 'before' or 'after'. Is required only if Image is specified)
+        }
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# Internal properties #}
+
+{% set _content_item = {
+  type: 'default',
+  description: '',
+  meta: [],
+  title: {},
+  image: '',
+  image_position: 'before',
+  links: []
+} %}
+
+{% if content_item is defined %}
+  {% set _content_item = _content_item|merge(content_item) %}
+{% endif %}
+
+{% set _css_class = 'ecl-content-item ecl-u-d-flex ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pv-m' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Create macro #}
+
+{% macro image(src, alt, class1, class2) %}
+  <div role="img" aria-label="{{ alt }}" class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-mr-xl {{ class1 }}"
+       style="background-image:url({{ src }});background-size:contain;height:5rem;width:7.5rem"></div>
+  <div role="img" aria-label="{{ alt }}"
+       class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-mr-xl ecl-u-d-none {{ class2 }}"
+       style="background-image:url({{ src }});background-size:contain;height:8.75rem;width:13.125rem">
```

Comment:
We didn't have to remove the single quotes, we have to make sure that whater character is passed in the URL does not break the HTML. For example, if I use this:

```yaml
    image:
      type: "array"
      label: "Primary image"
      description: "Attributes of the image to render"
      preview:
        src: "https://placeimg.com/1200/900/arch&test=')"
        alt: "Alternative text for image"
```

The the HTML gets rendered like that:

![image](https://user-images.githubusercontent.com/153362/62867867-9814b400-bd14-11e9-8dcb-de80c5d54850.png)

Which is clearly broken.

Reply:
Pushed

Reply:
So about the issue above we decided to open a follow up as we need to carefully tests all sorts of scenarios with this one. We will add a comment with a link to said follow up.